### PR TITLE
Common: add viewpro to upcoming features

### PR DESCRIPTION
**Pull request by rmackay9**
This adds a wiki page for the ViewPro gimbal support which will be included in 4.4.  The corresponding flight code PR is https://github.com/ArduPilot/ardupilot/pull/22892

I have tested this on my local machine and it looks fine and no errors.

### CODE REVIEW

File: common/source/docs/common-viewpro-gimbal.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+- Set :ref:`SERIAL2_PROTOCOL <SERIAL2_PROTOCOL>` = 28 (Scripting).  This assumes the gimbal is attached to the autopilot's Serial2 port but any serial port can be used.
+- Set :ref:`SCR_ENABLE <SCR_ENABLE>` = 1 to enable scripting and reboot the autopilot
+- Set :ref:`MNT1_TYPE <MNT1_TYPE>` = 9 (Scripting) to enable the mount/gimbal scripting driver
+- Set :ref:`CAM_TRIGG_TYPE <CAM_TRIGG_TYPE>` = 3 (Mount) to enable camera control using the mount driver
+- Download mount-viewpro-driver.lua (`from here <https://github.com/ArduPilot/ardupilot/tree/master/libraries/AP_Scripting/drivers>`__) and copy it to the autopilot's SD card in the APM/scripts directory and reboot the autopilot
+- Optionally for gimbals with multiple cameras, an auxiliary switch can control the active camera by doing the following
+  
+  - Set :ref:`RC9_OPTION <RC9_OPTION>` = 300 (Scripting1) to allow selecting the acdtive camera with RC9.  Note that any auxiliary switch may be used.
+  - VIEP_CAM_SWLOW, VIEP_CAM_SWMID, and VIEP_CAM_SWHIGH control the active camera when the aux switch is the low, medium and high position respectively.  Possible values are
```
Comment (Hwurzburg) [inline common/source/docs/common-viewpro-gimbal.rst:37]:
```suggestion
  - When the driver script is run, it creates three new parameters : VIEP_CAM_SWLOW, VIEP_CAM_SWMID, and VIEP_CAM_SWHIGH. These control the active camera when the aux switch is the low, medium and high position respectively.  Possible values are
```

File: common/source/docs/common-viewpro-gimbal.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      - 0: No change in camera selection
+      - 1: EO1
+      - 2: IR thermal
+      - 3: EO1 + IR Picture-in-picture
+      - 4: IR + EO1 Picture-in-picture
+      - 5: Fusion
+      - 6: IR1 13mm
+      - 7: IR2 52mm
+
+- Set VIEP_ZOOM_SPEED to control speed of zoom (value between 0 and 7)
```
Comment (Hwurzburg) [inline common/source/docs/common-viewpro-gimbal.rst:48]:
```suggestion
- Set VIEP_ZOOM_SPEED parameter created by the driver script to control the speed of zoom (value between 0 and 7)
```

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

**Comment by rmackay9** [comment]
The flight code PR has gone in now so maybe we can merge this?

**Comment by Hwurzburg** [comment]
done..thanks

**Pull request merged**